### DISCUSSION TITLE
Debug logging

### DISCUSSION THREAD

**Discussion by tacerus** [keycloak/keycloak#14643] [Archive]
Hi,

I could not find this upon searching in the documentation.
How to set the log level to `DEBUG` in a **non**-Docker installation?

The admin console of a single realm reports "Unexpected errors" while redirect looping and there is zero information on it with the default log level.

Thanks for any pointers!

**Comment by pohjamat** [comment]
Hi,
did you try this command line parameter or env variable?
CLI: --log-level
Env: KC_LOG_LEVEL
https://www.keycloak.org/server/all-config#_logging
regards,    Matti
